recon_linux.go

```diff
@@ -43,17 +44,16 @@ func (irc *IrcCon) StartUnixListener() {
 }
 
 // Attempt to hijack session previously running bot
-func (irc *IrcCon) HijackSession() bool {
+func (irc *Bot) hijackSession() bool {
 	unaddr, err := net.ResolveUnixAddr("unix", irc.unixastr)
 	if err != nil {
-		irc.Log(LWarning, "could not resolve unix socket")
+		irc.Error("could not resolve unix socket")
 		return false
 	}
 
 	con, err := net.DialUnix("unix", nil, unaddr)
 	if err != nil {
-		fmt.Println("Couldnt restablish connection, no prior bot.")
-		fmt.Println(err)
+		irc.Error("Couldnt restablish connection, no prior bot:", err)
```

Comment:
Yes, I just made it an error to easier see it in the logs while debugging. The whole recon stuff needs to be rewritten anyway, because you can't just send a TLS conn (it has a state ). No time to write more, but I have a plan for fixing it. I'll reply here later in the week. 

On August 30, 2015 12:50:06 AM CEST, Lars Gierth <notifications@github.com> wrote:
>>  		return false
>>  	}
>>  
>>  	con, err := net.DialUnix("unix", nil, unaddr)
>>  	if err != nil {
>> -		fmt.Println("Couldnt restablish connection, no prior bot.")
>> -		fmt.Println(err)
>> +		irc.Error("Couldnt restablish connection, no prior bot:", err)
>
>One could argue that this isn't an error, but a notice -- no prior bot
>being present is totally legit right?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/whyrusleeping/hellabot/pull/16/files#r38266750

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.